src/theme/ms24/index.ts

```diff
@@ -1,13 +1,14 @@
 import { extendTheme } from '@chakra-ui/react';
 
 import colors from './colors';
-import { basis, borderRadius, spacing } from '../shared';
+import { basis, borderRadius, size, spacing } from '../shared';
 
 export const theme = {
   ...basis,
   colors,
   spacing,
```

Comment:
Not really related to scope of this PR, but we also can have a look on this Badge:

![image](https://user-images.githubusercontent.com/3179244/170170854-3c68d40d-8387-48f2-8420-5132ce18a8d6.png)

If we define `spacing` like this:

```ts
const spacing = {
  xs: '0.25rem',
  sm: '0.5rem',
  md: '0.75rem',
  lg: '1rem',
  xl: '1.25rem',
  '2xl': '1.5rem',
  '3xl': '2rem',
  '4xl': '3rem',
};
```

Then the smallest value for spacing will be `0.25rem`, which means `4px`. Meanwhile, we need `2px` as the smallest. This should be discussed with UX as well, I think

Reply:
Nice observation.
It seems that figma is not in sync. I'll check with the designer to adjust the spacing with the tokens that we already have (he is back on Monday). Thanks